modules/monitors/README.md

```diff
@@ -0,0 +1,79 @@
+below.
+
+There are some optional additions to the monitor definition that are not
+part of the Datadog API schema. These are:
+- `force_delete` (boolean): If true, when deleting the monitor, the monitor 
+  will be deleted even if it is referenced by other resources.
+- `validate` (boolean): If false, the monitor will not be validated during
+  the Terraform plan phase.
+- `enabled` (boolean): If false, the monitor will not be created. This is to 
+  allow you to suppress a monitor created through merging configuration snippets.
+
+### Tags
+
+This module provides special handling for tags. The `tags` attribute of a monitor
+definition in the Datadog API and the Terraform resource is a list of strings. 
+However, the Cloud Posse (as well as AWS) default is to use a map of strings
+for tags. This module allows you to provide either a list or a map for the tags.
+
+If you provide a list, it will be used as is. If you provide a map, it will be
+converted to a list of strings in the format `key:value` (or `key` if `value` is
+`null`). If you provide no tag settings at all, not even an empty list or map,
+the module will use the default tags from the [null-label](https://github.com/cloudposse/terraform-null-label/) module.
+
+### API Schema (preferred)
+
+Datadog provides a REST API for managing monitors. We refer to the responses
+to the API requests for monitor definitions (`GET https://api.datadoghq.com/api/v1/monitor/{monitor_id}`) 
+as the "API Schema" for the tests. This should correspond to the documented 
+API "Model" for Monitors plus additional information such as creation date,
+but if the documentation and the API response differ, we use the API response
+as the source of truth.
+
+You can retrieve Monitor definitions from Datadog via the Datadog Web Console.
+Navigate to the monitor you want to retrieve, click the gear icon in the upper
+right (representing "settings"), and select "Export". This will display a JSON
+representation of the monitor definition. You can then click the "Copy" button
+to copy the JSON to the clipboard.
+
+You can then use the `jsondecode()` function in Terraform to convert the JSON
+to a Terraform object, and use that object as the value for the monitor definition.
+You can also transform the JSON to HCL other ways, however you prefer. The
+relevant point is that this module will accept the monitor definition in this
+schema. Any field in the API schema that does not have a counterpart in the
+Terraform schema will be ignored.
+
+#### Special Notes
+
+The `alert_tags` input is provided for convenience. It is used to add notification
+tags to the monitor message. However, it does not check to see if the tags are
+already present. If the tags are already present, they will still be added again.
+Thus, if you define a monitor via JSON, and then you use `alert_tags` when
+creating it, and then export the JSON representation of the created monitor definition, 
+it will not match because of the added tags.
```

Comment:
```suggestion
> [!WARNING]
> If you define a monitor via JSON, and then you use `alert_tags` when
> creating it, and then export the JSON representation of the created monitor 
> definition, it will not match because of the added tags.
```